tests/Microsoft.Fx.Portability.MetadataReader.Tests/ReflectionMetadataInfoTests.cs

```diff
@@ -46,7 +46,7 @@ public void UnresolvedAssemblyTest()
             Assert.Equal(s_expectedResult.Count(), actual.Count());
 
             // Use this instead of Assert.Equal so it will output the missing item
-            foreach (var items in s_expectedResult.Zip(actual, Tuple.Create))
+            foreach (var items in actual.Zip(s_expectedResult, Tuple.Create))
```

Comment:
Weird. If it turns out that they really are switched, we may want to dig in more since there could be an xunit bug or something worth discovering.